### DISCUSSION TITLE
jsbsim for UE Error when applying the A320 aircraft

### DISCUSSION THREAD

**Discussion by etiennelvyi** [JSBSim-Team/jsbsim#1071] [Q&A]
When i test the jsbsim for ue5, i found that if i switch the JSBSimMovement-Aircraft Model to A320(787-8,737,B747 have no problem), the AircraftState.ECEFLocation will be "nan" and UE will then crash  when trying to Apply the transform to the Parent actor.
Does anyone meet the same problem?
![图片](https://github.com/JSBSim-Team/jsbsim/assets/32133913/a0eb7048-abfa-4815-8efa-3462fbde110e)

**Comment by gallonmate** [comment]
Go back in the Call Stack and see which point (wheel gear, or body point/structural point) is causing the nan. You may need to rebuild JSBSim and UE5 as Debug in order to fiqure that out.
You can also go into the the A320 xml, there may be some unusual wheel, bogey, or structural point that is causing the nan. Compare those values to other aircraft xml and see if there is something usual. 

Another thing to check is move the aircraft higher off the ground and check try changing it's structural offset/origin in the level editor details panel for the blueprint.

I can check it's function in UE5 in a few days.

**Reply by etiennelvyi** [reply]
thank you very much i will check it as you suggested.
